syntax = "proto3";

message ChatRoomRequest {
  string type = 1;
  string user_name = 2;
  string content = 3;
  map<string,string> user_list = 4;
}